jmxtrans-output/jmxtrans-output-kafka/src/main/java/com/googlecode/jmxtrans/model/output/kafka/KafkaWriter.java

```diff
@@ -77,7 +77,7 @@ public KafkaWriter(
 		kafkaProperties.setProperty("zk.connect", Settings.getStringSetting(settings, "zk.connect", null));
 		kafkaProperties.setProperty("serializer.class", Settings.getStringSetting(settings, "serializer.class", null));
 		this.producer= new Producer<String,String>(new ProducerConfig(kafkaProperties));
-		this.topics = asList(topics.split(","));
+		this.topics = asList(Settings.getStringSetting(settings, "topics", null).split(","));
```

Comment:
CodeReviewHub uses "thumbs up" to mark comments as corrected (see [documentation](https://www.codereviewhub.com/) for details). So adding a :-1: to mark it as uncomplete ...